modality-probe-cli/src/log/mod.rs

```diff
@@ -335,14 +348,13 @@ fn print_as_graph<W: WriteIo>(
                     }
                     LogEntryData::TraceClock(lc) | LogEntryData::TraceClockWithTime(.., lc) => {
                         // This is a local clock.
-                        let (lc_id, clock) = lc.pack();
                         if lc.id == *probe_id {
                             // The self-clock `0` is useful from a
                             // causal standpoint, but from a
                             // coordinating-timelines standpoint, it's
                             // ambiguous because it appears whether a
                             // snapshot was produced or not.
-                            if clock != 0 {
+                            if lc.pack().1 != 0 {
```

Comment:
Will this logic be problematic for probes that implement restart tracking?

Reply:
IIUC, no, I don' t _think_ so. This clock is packed, so a `0` would be

```rust
LogicalClock {
    id,
    epoch: ProbeEpoch(0),
    ticks: ProbeTicks(0),
}
```